R/test_result.R

```diff
@@ -1,127 +1,143 @@
 #' Test the result of exercise code
 #'
-#' \code{test_result()} executes tests against the final result of the user code.
-#' If a test throws an error, the test fails and the submitted answer will be marked incorrect.
+#' \code{test_result()} executes tests against the final result of the user code,
+#' this is usually a function definition.
+#' This is similar to the instructor providing passing and failing unit tests of student
+#' written functions.
+#' If a test throws an error, the test fails and the submitted answer will be marked incorrect
+#' (i.e., a \code{\link{graded}} object with \code{correct = FALSE}).
 #'
-#' @param correct A character string to display if all tests pass.
-#'   This character string will be run through \code{glue::\link[glue]{glue_data}} with
-#' \itemize{
-#'   \item \code{num_correct}: Number of correct tests. (Equals \code{num_total})
-#'   \item \code{num_total}: Number of tests
-#'   \item \code{errors}: Vector of errors found. (\code{NULL})
-#' }
-#' @param incorrect A character string to display if at least one test fails.
-#'   This character string will be run through \code{glue::\link[glue]{glue_data}} with
-#' \itemize{
-#'   \item \code{num_correct}: Number of correct tests
-#'   \item \code{num_total}: Number of tests
-#'   \item \code{errors}: Vector of errors found
-#' }
+#' @param ... \code{\link{pass_if}} or \code{\link{fail_if}} \code{\link{condition}}s to check
+#' @template correct
+#' @template incorrect
 #' @template grader_args
 #' @template learnr_args
-#' @param ... ignored
+#' @param glue_correct A glue string that returns the final correct message displayed.
+#'    Defaults to \code{getOption("gradethis_glue_correct_test")}, e.g.,
+#'    2/2 correct! Absolutely fabulous!.
+#' @param glue_incorrect A glue string that returns the final correct message displayed.
+#'    Defaults to \code{getOption("gradethis_glue_correct_test")}, e.g.,
+#'    1/2 correct! Try it again; next time's the charm!.
```

Comment:
missing code and quotes.

`\code{"1/2 correct! ... "}`

Reply:
Will make the change in #68